apps/omg_eth/lib/omg_eth/root_chain/rpc.ex

```diff
@@ -20,7 +20,8 @@ defmodule OMG.Eth.RootChain.Rpc do
 
   def call_contract(client \\ Ethereumex.HttpClient, contract, signature, args) do
     data = signature |> ABI.encode(args) |> Encoding.to_hex()
-    client.eth_call(%{to: contract, data: data})
+    from = Application.fetch_env!(:omg_eth, :eth_call_from_address)
```

Comment:
this feels very ugly, accessing the configuration way to deep in the stack, where all calls coming from?

Reply:
In this case I decided to move it a level higher, where we access `OMG.Eth.Configuration`.